clients/osv/src/funTest/kotlin/OsvServiceFunTest.kt

```diff
@@ -50,14 +48,12 @@ private val VULNERABILITY_FOR_PACKAGE_BY_INVALID_COMMIT_REQUEST = Vulnerabilitie
 private fun Vulnerability.patchIgnorableFields() =
     copy(
         modified = Instant.EPOCH,
-        databaseSpecific = emptyJsonObject.takeIf { databaseSpecific != null },
+        databaseSpecific = null,
         affected = affected.mapTo(mutableSetOf()) { affected ->
-            affected.copy(ecosystemSpecific = emptyJsonObject.takeIf { affected.ecosystemSpecific != null })
+            affected.copy(ecosystemSpecific = null)
```

Comment:
Why has line 53 been changed?

Reply:
Nevermind, I misunderstood the intention of the logic I believe, and have restored this in favor of updating the expected results.